src/psyclone/tests/domain/lfric/lfric_config_test.py

```diff
@@ -130,10 +138,55 @@ def test_entry_not_bool(tmpdir, option):
         assert "Not a boolean: tree" in str(err.value)
 
 
+@pytest.mark.parametrize("option", ["NUM_ANY_SPACE",
+                                    "NUM_ANY_DISCONTINUOUS_SPACE"])
+def test_entry_not_int(tmpdir, option):
+    ''' Check that we raise an error if the value of any options expecting
+    an integer value is not int() with base 10. '''
+    content = re.sub(r"^{0} = .*$".format(option),
+                     "{0} = false".format(option),
+                     _CONFIG_CONTENT,
+                     flags=re.MULTILINE)
+    config_file = tmpdir.join("config_dyn")
```

Comment:
Given that this pattern is used a number of times in the tests, could it be turned into a utility function i.e. `def config(config_file, content):`. The call to this function could then be wrapped with `pytest.raises...`?

Reply:
Melikes the idea very much so I applied it here and in the `config_test.py`.